<?xml version="1.0" ?>
<launch>
  <!--
      Launch file for the admittance controller
  -->

  <arg name="TOPIC_ARM_STATE" default="/joint_torque_controller/joint_state"/>
  <arg name="TOPIC_ARM_COMMAND" default="/joint_torque_controller/command_joint_torque"/>
  <arg name="TOPIC_WRENCH_STATE" default="/wrench_fake"/>


  <param name="topic_arm_state" value="$(arg TOPIC_ARM_STATE)"/>
  <param name="topic_arm_command" value="$(arg TOPIC_ARM_COMMAND)"/>
  <param name="topic_wrench_state" value="$(arg TOPIC_WRENCH_STATE)"/>


  <rosparam file="$(find impedance)/config/ImpedanceParams.yaml" command="load"/>

  <node pkg="impedance" name="impedance" type="impedance_node" output="screen"/>
</launch>